targets/simple_switch/main.cpp

```diff
@@ -46,6 +46,9 @@ main(int argc, char* argv[]) {
   simple_switch_parser.add_uint_option(
       "drop-port",
       "Choose drop port number (default is 511)");
+  simple_switch_parser.add_uint_option(
+      "priority-queues",
+      "Number ofpriority queues (default is 1)");
```

Comment:
@duskmoon314 Could you add space between `of` and `priority`?

Reply:
Ok, in a second